plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -1170,4 +1175,12 @@ static boolean isUnixSocketUri(URI uri) {
     private void createTarArchive(File tar, File... files) throws IOException {
         TarUtils.tarFiles(tar, 0, files);
     }
+
+    /**
+     * Gets the Docker host ip address. This is host that can be reached from a docker container. {@link https://docs.docker.com/articles/networking/}
```

Comment:
`This is host that can be reached from a docker container` a lot of hosts can be reached from container. 
Maybe `This is address of host that can be reached from a docker container`?